### PR TITLE
Cleartext replies

### PR DESCRIPTION
**Pull request by Valodim**
I slightly changed the section about cleartext replies, removing the inline text suggestion and mentioning key gossip. I also moved it below the "Mail Encryption" headline.

### CODE REVIEW

**Comment by dkg** [comment]
I'm willing to accept with this change, though i would prefer to have an example of the "note" that should be "left".

File: doc/level1.rst [hunk 1]
```diff
@@ -527,6 +501,26 @@ partially composed messages to their e-mail provider (e.g., in the
...
+
+Cleartext replies to encrypted mail
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the common use case, a reply to an encrypted message will also be
+encrypted. Due to Autocrypt's opportunistic approach to key discovery,
+however, it is possible that keys for all recipients are not
+available, and a reply can only be sent in the clear.
+
+A MUA that is relatively more oriented towards privacy aware users MAY
```
Comment (dkg) [inline doc/level1.rst]: maybe "privacy-aware users" ?

File: doc/level1.rst [hunk 2]
```diff
@@ -527,6 +501,26 @@ partially composed messages to their e-mail provider (e.g., in the
...
+prepare the cleartext reply without including any of the quoted and
+attributed text from the previous message, to avoid leaking the
+cleartext from the original encrypted message. In this case, a note
+should be left that the content was redacted, to avoid confusing the
+user.
```
Comment (dkg) [inline doc/level1.rst]: Confusing the user who is composing the message is one thing.  The note might itself also be oriented toward the recipient of the message, right?  reducing their confusion is also a good rationale.

Comment (Valodim) [reply doc/level1.rst]: This depends on how the MUA can display this information, I imagine that many will have no other option than placing it inline.

Do you have a concrete change in mind here?

**Pull request merged**